From 24a759bf689d6c58bd3ece6fbdd0ec66f87cada8 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Fri, 14 Sep 2018 15:51:22 +0300
Subject: [PATCH] Makefile: Fix distclean

relate to belwo Jira issues to make distclean more robust
http://192.168.0.63:8080/browse/SW-1526
http://192.168.0.63:8080/browse/SW-1527
---

diff --git a/Makefile b/Makefile
index 9cd20f6..4b8324f 100644
--- a/Makefile
+++ b/Makefile
@@ -184,6 +184,7 @@
 .PHONY: goya-distclean gaudi-distclean
 goya-distclean gaudi-distclean:
 	@rm -f $(HL_VARS_FILE)
+	@rm -f $(OUTPUT_DIR)/.*config*
 	@if [ -d $(OUTPUT_DIR)/build ]; then \
 		cd $(OUTPUT_DIR)/build && \
 		rm -rf \
@@ -198,6 +199,8 @@
 hl-distclean:
 ifneq ($(HL_DEV),)
 	@$(MAKE) $(HL_DEV)-distclean
+else
+	@$(MAKE) goya-distclean gaudi-distclean
 endif
 
 .PHONY: goya-onix-test goya-pldm-test goya-asic-test
